implementations/rust/ockam/ockam/src/entity/profile/proof.rs

```diff
@@ -0,0 +1,82 @@
+use arrayref::array_ref;
+use serde::de::Visitor;
+use serde::{Deserialize, Deserializer, Serialize, Serializer};
+
+#[derive(Serialize, Deserialize, Debug, Clone, Copy, Eq, PartialEq)]
+pub enum SignatureType {
+    SelfSign,
+    Previous,
+}
+
+#[derive(Debug, Clone)]
+struct SignatureData([u8; 64]);
+
+impl Serialize for SignatureData {
```

Comment:
See https://crates.io/crates/serde-big-array

Reply:
tnx, I wish I knew that